docs/reference/nuget-config-file.md

```diff
@@ -113,9 +113,9 @@ Note that the source URL for nuget.org is `https://api.nuget.org/v3/index.json`.
 
 Lists all known package sources. The order is ignored during restore operations and with any project using the PackageReference format. NuGet respects the order of sources for install and update operations with projects using `packages.config`.
 
-| Key | Value | protocolVersion |
-| --- | --- | --- |
-| (name to assign to the package source) | The path or URL of the package source. | The NuGet server protocol version to be used. The current version is "3". Defaults to version "2" when not pointing to a package source URL ending in `.json` (e.g. https://api.nuget.org/v3/index.json). Supported in [NuGet 3.0+](https://learn.microsoft.com/nuget/release-notes/nuget-3.0.0). See [NuGet Server API](https://learn.microsoft.com/nuget/api/overview) for more information about the version 3 protocol. |
+| Key | Value | protocolVersion | allowInsecureConnections |
+| --- | --- | --- | --- |
+| (name to assign to the package source) | The path or URL of the package source. | The NuGet server protocol version to be used. The current version is "3". Defaults to version "2" when not pointing to a package source URL ending in `.json` (e.g. https://api.nuget.org/v3/index.json). Supported in [NuGet 3.0+](https://learn.microsoft.com/nuget/release-notes/nuget-3.0.0). See [NuGet Server API](https://learn.microsoft.com/nuget/api/overview) for more information about the version 3 protocol. | The flag to indicate if allowing insecure/HTTP source for this package source.  Users will see HTTPS warnings/errors for an 'HTTP' source since NuGet 6.3, with setting `allowInsecureConnections` to true for this 'HTTP' source, NuGet will not show HTTPS warnings/errors for this 'HTTP' source. Defaults to "false" when `allowInsecureConnections` is not specified. Supported in NuGet 6.8+. |
```

Comment:
```suggestion
| (name to assign to the package source) | The path or URL of the package source. | The NuGet server protocol version to be used. The current version is "3". Defaults to version "2" when not pointing to a package source URL ending in `.json` (e.g. https://api.nuget.org/v3/index.json). Supported in [NuGet 3.0+](https://learn.microsoft.com/nuget/release-notes/nuget-3.0.0). See [NuGet Server API](https://learn.microsoft.com/nuget/api/overview) for more information about the version 3 protocol. | The flag to indicate if insecure/HTTP connections are allowed for this package source.  If you see HTTP usage warnings, setting `allowInsecureConnections` to true for this 'HTTP' source, will disable those warnings permanently. Defaults to "false" when `allowInsecureConnections` is not specified. Supported in NuGet 6.8+. |
```

I'm not super confident in this one. 
I think the description can be shorter than maybe  what I wrote. 

Maybe @donnie-msft or @zivkan can help.

Reply:
Thanks for reviewing! I pivoted the table, and used the shorter description.